app/knights.py

```diff
@@ -0,0 +1,45 @@
+from app.ammunition.armour import Armour
+from app.ammunition.potion import Potion
+from app.ammunition.weapon import Weapon
+
+
+class Knight:
+    """"Knight transformed from incoming dict key to class"""
+    def __init__(self,
```

Comment:
Apply next line for each paramter so they look like this https://stackoverflow.com/a/41591509

Reply:
Did this, but now code doesn't look better, as for me.